inventory/hosts.example

```diff
@@ -37,6 +53,23 @@ openshift_deployment_type=origin
 # release.
 openshift_release=v3.7
 
+# default subdomain to use for exposed routes, you should have wildcard dns
+# for *.apps.test.example.com that points at your infra nodes which will run
+# your router
+openshift_master_default_subdomain=apps.test.example.com
+
+#Set cluster_hostname to point at your load balancer
+openshift_master_cluster_hostname=ose-lb.test.example.
```

Comment:
this should be "ose-lb.test.example.com", right? When the cluster comes up it's trying to check status on "ose-lb.test.example." and that continuously fails.

Reply:
yep, updated.